app/controllers/PingController.php

```diff
@@ -128,7 +128,7 @@ public function pingPost()
 			
 			Log::debug("Created ".$param['id']." from ".$_SERVER['REMOTE_ADDR']);
 		}
-		return 'SUCCES';
+		return 'SUCCESS';
```

Comment:
@ShekharReddy4 very good question. Though i would understand why he may not have done so for such an obvious typo fix. :)